app/services/csv/append_notes_content.rb

```diff
@@ -1,26 +1,33 @@
 class Csv::AppendNotesContent
-  def call(csv, data)
+  def call(csv, notes)
     append_headings(csv)
-    append_notes(csv, data)
+    append_notes(csv, notes)
   end
 
   private
 
   def append_headings(csv)
-    csv << %w(Note Time Latitude Longitude)
+    csv << %w(Note Time Latitude Longitude Photo_Url)
   end
 
-  def append_notes(csv, data)
-    data.notes.each do |note|
+  def append_notes(csv, notes)
+    notes.each do |note|
       append_note(csv, note)
     end
   end
 
   def append_note(csv, note)
-    csv << [note["text"], format_time(note["date"]), note["latitude"], note["longitude"]]
+    csv << [note.text, format_time(note.date), note.latitude, note.longitude, format_url(note)]
   end
 
   def format_time(date)
     date.strftime("%FT%T")
   end
+
+  def format_url(note)
+    return "No photo was attached" unless note.photo_file_name
+
+    current_host_url = Rails.application.config.secrets.fetch('HOST_URL')
+    current_host_url.to_s + note.photo.url
```

Comment:
Yes. That is why I didn't add a `/` at the end of HOST_URL, but now I'm thinking that maybe it's better to have it in both places. The url will work fine with two slashes. And just in case someone will use `HOST_URL` in another place, it's safer if it contains the final `/`.